test/common.ts

```diff
@@ -0,0 +1,7 @@
+export function toNormalizedJsonString(payload: object) {
+  // GitHub sends its JSON with an indentation of 2 spaces and a line break at the end
+  const payloadString = JSON.stringify(payload, null, 2) + "\n";
```

Comment:
This information isn't correct. It never was :man_shrugging: 
```suggestion
  // GitHub sends its JSON with no indentation and no line breaks at the end
  const payloadString = JSON.stringify(payload);
```

Reply:
Would require new hashes for all the tests. Maybe not for now